LecturesScheduler.WebApi/Startup.cs

```diff
@@ -101,7 +99,13 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env)
 
             app.UseMiddleware<ErrorHandlingMiddleware>();
             app.UseHttpsRedirection();
-            app.UseMvc();
-        }       
+            app.UseRouting();
+
+            //app.UseMvc();
```

Comment:
?

Reply:
Su .UseMvc() gaudau error
`using 'UseMvc' to configure MVC is not supported while using Endpoint Routing. To continue using 'UseMvc', please set 'MvcOptions.EnableEndpointRouting = false' inside 'ConfigureServices'.`
pagal šitą article apie migravimą į .net nebereikia naudoti .UseMvc, o jį pakeisti į .UseEndpoints. 
https://docs.microsoft.com/en-us/aspnet/core/migration/22-to-30?view=aspnetcore-3.0&tabs=visual-studio